src/components/NakedButton.tsx

```diff
@@ -1,13 +1,24 @@
-import styled from "@emotion/styled"
+import React from "react"
+import { Button, ButtonProps } from "@chakra-ui/react"
+
+const NakedButton: React.FC<ButtonProps> = ({ children, ...props }) => (
```

Comment:
Have you check the `ghost` variant? https://chakra-ui.com/docs/components/button/usage#button-variants

I think we should remove this component and start using `<Button variant="ghost">`. Any specific style we want to do on top of the default variant styles, we could add them on the Chakra theme definition as we did with the other variants.

Check the `variants` key in [here](https://github.com/ethereum/ethereum-org-website/blob/dev/src/%40chakra-ui/gatsby-plugin/components/Button.ts).